common/spdat.cpp

```diff
@@ -776,6 +776,21 @@ bool IsResurrectionEffects(uint16 spell_id)
 	return false;
 }
 
+int8 DoResSicknessChecks(uint16 spell_id1, uint16 spell_id2)
+{
+	if (RuleI(Spells, ResurrectionEffectBlock) == 1) {
```

Comment:
Maybe turn these magic numbers into constants? Plenty of examples around.

Reply:
Curious as to how you'd like these done. I get where some things that are recycled are done as constants, just not sure how you'd like it done here.